packages/web-api/src/controllers/health-check-controller.ts

```diff
@@ -13,14 +15,30 @@ export class HealthCheckController extends ApiController {
     public constructor(
         @inject(ServiceConfiguration) protected readonly serviceConfig: ServiceConfiguration,
         @inject(Logger) logger: Logger,
+        @inject(webApiTypeNames.ApplicationInsightsClientProvider)
+        protected readonly appInsightsClientProvider: ApplicationInsightsClientProvider,
     ) {
         super(logger);
     }
 
     public async handleRequest(): Promise<void> {
         this.logger.trackEvent('HealthCheck');
```

Comment:
I'm a bit concerned about querying app insights in the health check end point. 
Feel free to correct me: if app insights is down, would the health check throw error? If so health check api will return 404 but our service is actually alive.

Reply:
Good point, I'll add a check to handle those errors